pkg/kubewarden/modules/policyReporter.ts

```diff
@@ -195,43 +168,27 @@ export function getLinkForPolicy(store: any, report: PolicyReportResult): Object
     const apSchema = store.getters['cluster/schemaFor'](KUBEWARDEN.ADMISSION_POLICY);
     const capSchema = store.getters['cluster/schemaFor'](KUBEWARDEN.CLUSTER_ADMISSION_POLICY);
 
-    /**
-     * Split policy name as it is formatted like: 'cap-do-not-run-as-root'
-     * To determine which type of policy we need the first part of the string,
-     * 'cap' or 'ap' for ClusterAdmissionPolicy and AdmissionPolicy respectively.
-     */
     let policyType: string = '';
-    let policyParts: string[] = report.policy.split('-');
-    let policyNs: string | null = null;
-
-    if ( policyParts.length >= 2 ) {
-      policyParts = [policyParts[0], policyParts.slice(1).join('-')];
-      policyType = policyParts[0] === 'cap' ? KUBEWARDEN.CLUSTER_ADMISSION_POLICY : KUBEWARDEN.ADMISSION_POLICY;
 
-      // Find the namespace of the AdmissionPolicy
-      if ( policyType === KUBEWARDEN.ADMISSION_POLICY && apSchema ) {
-        const admissionPolicies = store.getters['cluster/all'](KUBEWARDEN.ADMISSION_POLICY);
-        const resource = admissionPolicies?.find((ap: any) => ap.metadata?.name === policyParts[1]);
+    const policyParts: string[] = report.policy?.split('-');
 
-        if ( resource ) {
-          policyNs = resource.metadata?.namespace;
-        }
-      }
+    if (policyParts?.length >= 2 ) {
```

Comment:
now the PolicyReportResults should have `properties.policy-namespace` when the policy is a namespaced one (AdmissionPolicy). I would prefer to use that to filter, instead of parsing the name.

Reply:
That's true @viccuad .  Either I got it wrong (too much stuff going on at the same time), but I was only counting on `properties.policy-name`. This simplifies the code. Thanks for the heads up 🙏